feature-utils/poly-analysis/src/index.js

```diff
@@ -1,8 +1,8 @@
 import {
-    UnrecognizedData,
+    analyzeZip,
     runAnalysis,
     analyzeFile,
-    analyzeZip,
+    UnrecognizedData,
```

Comment:
Er. Why the swap?

Reply:
just getting the shortest text on top and longest on the bottom, nothing other that

Reply:
Nice :-)